test/verify/check-services

```diff
@@ -192,11 +192,18 @@ Unit=test.service
         self.wait_service_state("test.service", "inactive")
         self.wait_service_in_panel("test.service", "Enabled")
 
+        if self.machine.image != "rhel-8-2-distropkg":
+            b.wait_attr("#services-toolbar", "data-loading", "false")
+
         # Selects Targets tab
         self.pick_tab(2)
         b.wait_not_in_text("#services-list", "test")
         self.wait_service_state("sockets.target", "active")
+
+        # Make sure that symlinked services also appear in the list
         self.wait_service_state("reboot.target", "inactive")
+        self.wait_service_state("ctrl-alt-del.target", "inactive")
+        self.wait_service_state("runlevel6.target", "inactive")
```

Comment:
These two [also need to be conditionalized on distropkg](https://logs.cockpit-project.org/logs/pull-13984-20200428-165709-357a83d4-rhel-8-2-distropkg/log.html#258-2)

Reply:
done myself